tests/test_make.py

```diff
@@ -84,10 +95,12 @@ def fixture_from_dict(
     """
 
     @pytest.fixture(name=name, params=id_to_value.values(), ids=id_to_value.keys())
-    def fixture(request):
+    def fixture(request: _pytest.fixtures.SubRequest) -> object:
```

Comment:
What else are you thinking of?  `Any`?  I guess theoretically it could be a passthrough `TypeVar` from the `dict`, but that is kind of moot since `_FixtureFunction` isn't generic so that can't make it out of this function anyways.  And, as a second layer, fixture types don't make their way to their usages anyways.

Side note, I noticed that `_FixtureFunction` has been deleted so I guess we'll get to figure that out for pytest 6.3.